src/main/java/cr0s/warpdrive/WarpDrive.java

```diff
@@ -197,19 +203,19 @@ public void preInit(FMLPreInitializationEvent event) {
 		
 		logger = event.getModLog();
 		
-		if (FMLCommonHandler.instance().getSide().isClient()) {
-			Minecraft mc = Minecraft.getMinecraft();
-
-			normalFOV = mc.gameSettings.fovSetting;
-			normalSensitivity = mc.gameSettings.mouseSensitivity;
-			logger.info("FOV is " + normalFOV + " Sensitivity is " + normalSensitivity);
-		}
+		// Not needed as we register this at right click on the monitor.
+		/* if (FMLCommonHandler.instance().getSide().isClient()) {
+		 * 	Minecraft mc = Minecraft.getMinecraft();
+		 * 
+		 * 	normalFOV = mc.gameSettings.fovSetting;
+		 * 	normalSensitivity = mc.gameSettings.mouseSensitivity;
+		 * 	logger.info("FOV is " + normalFOV + " Sensitivity is " + normalSensitivity);
+		 * }
+		 */
```

Comment:
For now, can you just remove this change? I would like to take a look at it myself done it is somewhat contested, so if you remove it from this pull request I'll merge the rest right now, then we can take a look at this part again